client/notifications/activity-bar-entry.tsx

```diff
@@ -84,6 +87,7 @@ export function NotificationsButton() {
           icon={<NotificationsIcon />}
           title={hasUnread ? 'Notifications (unread)' : 'Notifications'}
           onClick={onClick}
+          hotkey={{ keyCode: KEY_N, altKey: true }}
```

Comment:
probably better to pull the entire hotkey config into a file-level constant tbh, I think done this way it will result in it changing every render (and isn't really providing value, since you can just replace the `KEY_N` constant above with something like: `const ALT_N: HotkeyProp = { keyCode: keycode('n'), altKey: true }`

Probably apply that everywhere these are used